packages/ember-glimmer/lib/component.js

```diff
@@ -109,13 +108,489 @@ export const BOUNDS = symbol('BOUNDS');
+  HTML associated attribute:
+
+  ```app/components/my-anchor.js
+  import Ember from 'ember';
+
+  export default Ember.Component.extend({
+    tagName: 'a',
+    attributeBindings: ['href'],
+    href: 'http://google.com'
+  });
+  ```
+  Will result in component instances with an HTML representation of:
+
+  ```html
+  <a id="ember1" class="ember-view" href="http://google.com"></a>
+  ```
+  One property can be mapped on to another by placing a ":" between
+  the source property and the destination property:
+
+  ```app/components/my-anchor.js
+  import Ember from 'ember';
+
+  export default Ember.Component.extend({
+    tagName: 'a',
+    attributeBindings: ['url:href'],
+    url: 'http://google.com'
+  });
+  ```
+  Will result in component instances with an HTML representation of:
+
+  ```html
+  <a id="ember1" class="ember-view" href="http://google.com"></a>
+  ```
+  Namespaced attributes (e.g. `xlink:href`) are supported, but have to be
+  mapped, since `:` is not a valid character for properties in Javascript:
+
+  ```app/components/my-use.js
+  import Ember from 'ember';
+
+  export default Ember.Component.extend({
+    tagName: 'use',
+    attributeBindings: ['xlinkHref:xlink:href'],
+    xlinkHref: '#triangle'
+  });
+  ```
+  Will result in component instances with an HTML representation of:
+
+  ```html
+  <use xlink:href="#triangle"></use>
+  ```
+  If the return value of an `attributeBindings` monitored property is a boolean
+  the attribute will be present or absent depending on the value:
+
+  ```app/components/my-text-input.js
+  import Ember from 'ember';
+
+  export default Ember.Component.extend({
+    tagName: 'input',
+    attributeBindings: ['disabled'],
+    disabled: false
+  });
+  ```
+  Will result in a component instance with an HTML representation of:
+
+  ```html
+  <input id="ember1" class="ember-view" />
+  ```
+  `attributeBindings` can refer to computed properties:
+
+  ```app/components/my-text-input.js
+  import Ember from 'ember';
+
+  export default Ember.Component.extend({
+    tagName: 'input',
+    attributeBindings: ['disabled'],
+    disabled: Ember.computed(function() {
+      if (someLogic) {
+        return true;
+      } else {
+        return false;
+      }
+    })
+  });
+  ```
+  To prevent setting an attribute altogether, use `null` or `undefined` as the
+  return value of the `attributeBindings` monitored property:
+
+  ```app/components/my-text-input.js
+  import Ember from 'ember';
+
+  export default Ember.Component.extend({
+    tagName: 'form',
+    attributeBindings: ['novalidate'],
+    novalidate: null
+  });
+  ```
+  Updates to the property of an attribute binding will result in automatic
+  update of the  HTML attribute in the component's rendered HTML representation.
+  `attributeBindings` is a concatenated property. See [Ember.Object](/api/classes/Ember.Object.html)
+  documentation for more information about concatenated properties.
+
+
+  ## Layouts
```

Comment:
This section needs some rewording due to codebase changes. Approach is being discussed in Slack.